### ISSUE TITLE
Uniform handling of points at infinity in biggroup

### ISSUE THREAD

**Issue by codygunton** [AztecProtocol/barretenberg#871]
The Goblin prover constructor mocks the interaction of a first circuit with the op queue due to the inability to currently handle zero commitments which would otherwise appear in the first round of the merge protocol. To be removed once the issue has been resolved. Edit: this may be resolved. Needs investigation.

**Comment by ledwards2225** [comment]
@federicobarbacovi I think you've confirmed that this is no longer an issue, right? We do not do any mocking to avoid an empty initial `T_prev`

**Comment by federicobarbacovi** [comment]
> [@federicobarbacovi](https://github.com/federicobarbacovi) I think you've confirmed that this is no longer an issue, right? We do not do any mocking to avoid an empty initial `T_prev`

Yes, correct. There is no mocking done in `ClientIVC`, `T_prev` is empty for the first merge.
